Online distro fails installing demo feature

On the first launch, the online distro shows these errors in the log file:
```
00:58:04.505 [INFO ] [.openhab.core.internal.CoreActivator] - openHAB runtime has been started (v2.0.0, build 201601082210).
00:58:53.899 [WARN ] [url.mvn.internal.AetherBasedResolver] - Error resolving artifactorg.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT:Could not find artifact org.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT in oh-snapshot-repo (http://oss.jfrog.org/libs-snapshot/)
shaded.org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact org.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT in oh-snapshot-repo (http://oss.jfrog.org/libs-snapshot/)
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:444)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts(DefaultArtifactResolver.java:246)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifact(DefaultArtifactResolver.java:223)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifact(DefaultRepositorySystem.java:294)[4:org.ops4j.pax.url.mvn:2.4.3]
	at org.ops4j.pax.url.mvn.internal.AetherBasedResolver.resolve(AetherBasedResolver.java:573)[4:org.ops4j.pax.url.mvn:2.4.3]
	at org.ops4j.pax.url.mvn.internal.AetherBasedResolver.resolve(AetherBasedResolver.java:528)[4:org.ops4j.pax.url.mvn:2.4.3]
	at org.ops4j.pax.url.mvn.internal.AetherBasedResolver.resolve(AetherBasedResolver.java:506)[4:org.ops4j.pax.url.mvn:2.4.3]
	at org.ops4j.pax.url.mvn.internal.AetherBasedResolver.resolve(AetherBasedResolver.java:481)[4:org.ops4j.pax.url.mvn:2.4.3]
	at org.ops4j.pax.url.mvn.internal.Connection.getInputStream(Connection.java:123)[4:org.ops4j.pax.url.mvn:2.4.3]
	at java.net.URL.openStream(URL.java:1038)[:1.8.0_40]
	at org.apache.karaf.features.internal.service.FeatureConfigInstaller.installConfigurationFile(FeatureConfigInstaller.java:210)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.FeatureConfigInstaller.installFeatureConfigs(FeatureConfigInstaller.java:120)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.installFeature(FeaturesServiceImpl.java:1156)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.Deployer.deploy(Deployer.java:769)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.doProvision(FeaturesServiceImpl.java:1079)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl$1.call(FeaturesServiceImpl.java:975)[9:org.apache.karaf.features.core:4.0.3]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)[:1.8.0_40]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)[:1.8.0_40]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)[:1.8.0_40]
	at java.lang.Thread.run(Thread.java:745)[:1.8.0_40]
Caused by: shaded.org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT in oh-snapshot-repo (http://oss.jfrog.org/libs-snapshot/)
	at shaded.org.eclipse.aether.connector.basic.ArtifactTransportListener.transferFailed(ArtifactTransportListener.java:39)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.connector.basic.BasicRepositoryConnector$TaskRunner.run(BasicRepositoryConnector.java:355)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.util.concurrency.RunnableErrorForwarder$1.run(RunnableErrorForwarder.java:67)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.connector.basic.BasicRepositoryConnector$DirectExecutor.execute(BasicRepositoryConnector.java:581)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.connector.basic.BasicRepositoryConnector.get(BasicRepositoryConnector.java:249)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads(DefaultArtifactResolver.java:520)[4:org.ops4j.pax.url.mvn:2.4.3]
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:421)[4:org.ops4j.pax.url.mvn:2.4.3]
	... 19 more
00:58:53.905 [ERROR] [core.karaf.internal.FeatureInstaller] - Failed installing 'openhab-package-demo': Error resolving artifact org.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT: Could not find artifact org.openhab.distro:openhab-demo-resources:cfg:services-basicui:2.0.0-SNAPSHOT in oh-snapshot-repo (http://oss.jfrog.org/libs-snapshot/)
```
I assume this is due to https://github.com/openhab/openhab-distro/pull/6, which does not deploy these files anymore.

--- a/features/openhab-demo-resources/pom.xml
+++ b/features/openhab-demo-resources/pom.xml
@@ -47,7 +47,13 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>false</skip>
+                </configuration>
+            </plugin>
         </plugins>
     </build>
-
 </project>

